torchvision/transforms/functional.py

```diff
@@ -728,14 +728,20 @@ def _get_inverse_affine_matrix(center, angle, translate, scale, shear):
     # Thus, the inverse is M^-1 = C * RSS^-1 * C^-1 * T^-1
 
     angle = math.radians(angle)
-    shear = math.radians(shear)
+    print(shear)
```

Comment:
Can you remove this print?

Reply:
Sure! Haven't noticed this debug statement.